projet/users/serializers.py

```diff
@@ -23,6 +23,12 @@ class Meta:
         fields = ("id", "email", "password", "image")
 
 
+class UserBulkSerializer(serializers.Serializer):
```

Comment:
Needed to create this serializer because once we remove the password field from the user serializer it throws errors and will not be able to create users because the password is a required field even in the models.